src/Updates.elm

```diff
@@ -0,0 +1,148 @@
+module Updates exposing (update)
+
+import Alerts.Api
+import Navigation
+import Silences.Types exposing (nullSilence)
+import Task
+import Types exposing (Msg(..), Model, Route(NotFoundRoute, SilenceFormEditRoute, SilenceFormNewRoute, SilenceRoute, StatusRoute, SilenceListRoute, AlertsRoute))
```

Comment:
Not sure what the convention is, but it might be nice to put this on multiple lines